applications/evaluate_mlp.py

```diff
@@ -112,7 +145,7 @@ def evaluate(conf, reevaluate=False):
     plot_confusion_matrix(
         data,
         labels,
-        normalize=True,
+        normalize="true",
```

Comment:
Why change from bool to string? Bool is more appropriate for True/False flags.

Reply:
There is a bug somewhere in the data loader script ... I havent dug it up yet but will fix this on the next commit.

Reply:
Ok the root of this is due to passing normalize directly into the sklearn.metrics.confusion_matrix, which wants None or 'true'. I've just updated our script so that its True and not 'true' and handle it in the method in plotting.py